software/__init__.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
+
+
+import os
+import sys
+
+LIB_PATHS = ('software/util', 'software/scripts', 'software/SchemaTerms', 'software/SchemaExamples')
+DATA_PATHS = ('docs', 'software/gcloud','data')
+_INITIALIZED = None
+
+def Setup():
+    """Setup the import path for the project and check the validity of the runtime."""
+    global _INITIALIZED
+    if _INITIALIZED:
+        return
+
+    if not (sys.version_info.major == 3 and sys.version_info.minor > 5):
+        sys.stderr.write("Python version %s.%s not supported version 3.6 or above required - Exiting" % (sys.version_info.major,sys.version_info.minor))
+        sys.exit(os.EX_CONFIG)
+
+    for path in LIB_PATHS:
+        absolute_path = os.path.join(os.getcwd(), path)
+        if not os.path.isdir(absolute_path):
+            sys.stderr.write('Required directory "%s" not found - Exiting\n' % absolute_path)
+            sys.exit(os.EX_CONFIG)
+        sys.path.insert(1, absolute_path)
+
+    _INITIALIZED = True
+
+
+def CheckWorkingDirectory():
```

Comment:
Did you intend to call this after `Setup()`? I can't see it referenced

Reply:
I meant to call it from the buildsite.py file. Fixed.